dask/dataframe/csv.py

```diff
@@ -40,7 +40,15 @@ def bytes_read_csv(b, header, kwargs, dtypes=None, columns=None,
         bio.write(header)
     bio.write(b)
     bio.seek(0)
-    df = pd.read_csv(bio, **kwargs)
+
+    try:
+        df = pd.read_csv(bio, **kwargs)
+    except (csv.Error, pd.io.common.CParserError) as e:
+        raise ValueError(
+            "There was an error in parsing a CSV block. This may be due to "
+            "the line terminator appearing in a quoted value. Original error: "
+            "%s" % str(e))
```

Comment:
If there isn't an easy way to identify the endline-in-quotes situation, then yes, probably.